types/ember-data/index.d.ts

```diff
@@ -892,11 +892,9 @@ export namespace DS {
      * A `ManyArray` is a `MutableArray` that represents the contents of a has-many
      * relationship.
      */
-    interface ManyArray<T> extends Ember.MutableArray<T> {}
-    class ManyArray<T> extends Ember.Object.extend(
-        Ember.MutableArray as {},
-        Ember.Evented
-    ) {
+    // tslint:disable-next-line:no-empty-interface -- used for declaration merge
+    interface ManyArray<T> extends Ember.MutableArray<T>, Evented {}
```

Comment:
Does tslint's `no-empty-interface` not allow empty interfaces that extend multiple supertypes like [@typescript-eslint/eslint-plugin's `no-empty-interface` does](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-empty-interface.md#-correct)?